src/commentsRequest.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line no-unused-vars
+const getItem = async (id) => {
+  const url = (`https://www.themealdb.com/api/json/v1/1/lookup.php?i=${id}`);
+  const response = await fetch(url);
+  const parsed = await response.json();
+  return parsed.meals[0];
+};
+
+/*
+const getComments = async (id) => {
```

Comment:
To keep the code clean, I would remove the commented code from the commits on each pull request